test_cases/test_dojo.py

```diff
@@ -43,4 +45,34 @@ def test_create_room_returns_error_message_if_roomname_not_string(self):
 		with self.assertRaises(ValueError):
 			self.the_dojo.create_room("OFFICE",123)
 	
+	def test_that_a_room_exists_before_printing_occupants(self):
+		self.the_dojo.create_room("office","Green")
+		self.assertTrue(self.the_dojo.print_room("Green"))
+
+	def test_print_room_returns_error_message_if_room_doesnot_exist(self):
+		self.the_dojo.create_room("office","blue")
+		self.assertEqual(self.the_dojo.print_room("red"), "This room doesnot exist!")
+
+	def test_print_room_prints_all_occupants_if_room_is_office(self):
+		self.the_dojo.create_room("office","Purple")
+		self.the_dojo.add_person("Pete","pete@pete","Staff")
+		self.assertEqual(self.the_dojo.print_room("Purple"), "Pete")
+
+	def test_print_room_prints_all_occupants_if_room_is_livingspace(self):
+		self.the_dojo.create_room("livingspace", "Ruby")
+		self.the_dojo.add_person("Marie", "marie@marie.com","Fellow", "y")
+		self.assertEqual(self.the_dojo.print_room("Ruby"), "Marie")
+
+	def test_if_prints_allocated_filepath_exists(self):
+		self.the_dojo.print_allocations("file1")
+		self.assertTrue(os.path.isfile("file1.txt"))
+		os.remove("file1.txt")
+
+	def test_it_print_unallocated_filepath_exists(self):
+		self.the_dojo.print_unallocated("file2")
+		self.assertTrue(os.path.isfile("file2.txt"))
+		os.remove("file2.txt")
```

Comment:
These will test whether the file was created. How will you know whether the content was actually written in there?